tests/node/component-rendering-test.js

```diff
@@ -44,6 +44,17 @@ QUnit.test("Component with dynamic value", function(assert) {
   assert.ok(html.match(/<h1>Hello World<\/h1>/));
 });
 
+QUnit.test("Component with dynamic safe attribute undefined", function(assert) {
```

Comment:
I understand what you are doing here (because I read through the HTMLBars PR), but this test title doesn't seem very descriptive.  Maybe something like:

> ensure undefine attributes requiring protocol sanitization do not error